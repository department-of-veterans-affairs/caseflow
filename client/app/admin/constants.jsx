export const ACTIONS = {
  SET_FEATURE_TOGGLES: 'SET_FEATURE_TOGGLES',
  POST_VETERAN_EXTRACT: 'POST_VETERAN_EXTRACT',
  STARTED_VETERAN_EXTRACT: 'STARTED_VETERAN_EXTRACT',
  FAILED_VETERAN_EXTRACT: 'FAILED_VETERAN_EXTRACT'
};
