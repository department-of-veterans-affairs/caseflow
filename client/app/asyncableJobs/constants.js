export const ACTIONS = {
  MANUAL_JOB_STARTED: 'MANUAL_JOB_STARTED',
  MANUAL_JOB_FAILED: 'MANUAL_JOB_FAILED',
  CLEAR_MANUAL_JOB_TYPE: 'CLEAR_MANUAL_JOB_TYPE',
};