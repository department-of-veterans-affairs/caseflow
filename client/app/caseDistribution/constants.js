import DISTRIBUTION from '../../constants/DISTRIBUTION';

const LEVERS = 'levers';

export const Constant = {
  LEVERS
};

export const sectionTitles = {
  [DISTRIBUTION.ama_hearings_start_distribution_prior_to_goals]: DISTRIBUTION.ama_hearings_section_title,
  [DISTRIBUTION.ama_direct_review_start_distribution_prior_to_goals]: DISTRIBUTION.ama_direct_review_section_title,
  [DISTRIBUTION.ama_evidence_submission_start_distribution_prior_to_goals]: DISTRIBUTION.ama_evidence_submission_section_title
};

export const docketTimeGoalPriorMappings = {
  [DISTRIBUTION.ama_hearings_start_distribution_prior_to_goals]:
    DISTRIBUTION.ama_hearings_docket_time_goals,
  [DISTRIBUTION.ama_direct_review_start_distribution_prior_to_goals]:
    DISTRIBUTION.ama_direct_review_docket_time_goals,
  [DISTRIBUTION.ama_evidence_submission_start_distribution_prior_to_goals]:
    DISTRIBUTION.ama_evidence_submission_docket_time_goals
};
