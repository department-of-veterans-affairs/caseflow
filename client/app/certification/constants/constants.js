// actions
//
export const UPDATE_PROGRESS_BAR = 'UPDATE_PROGRESS_BAR';
export const TOGGLE_CANCELLATION_MODAL = 'TOGGLE_CANCELLATION_MODAL';
export const CHANGE_VBMS_HEARING_DOCUMENT = 'CHANGE_VBMS_HEARING_DOCUMENT';
export const CHANGE_TYPE_OF_FORM9 = 'CHANGE_TYPE_OF_FORM9';
export const CHANGE_TYPE_OF_HEARING = 'CHANGE_TYPE_OF_HEARING';

export const CHANGE_REPRESENTATIVE_NAME = 'CHANGE_REPRESENTATIVE_NAME';
export const CHANGE_REPRESENTATIVE_TYPE = 'CHANGE_REPRESENTATIVE_TYPE';
export const CHANGE_OTHER_REPRESENTATIVE_TYPE = 'CHANGE_OTHER_REPRESENTATIVE_TYPE';

export const CHANGE_SIGN_AND_CERTIFY_FORM = 'CHANGE_SIGN_AND_CERTIFY_FORM';

export const CERTIFICATION_UPDATE_REQUEST = 'CERTIFICATION_UPDATE_REQUEST';
export const CERTIFICATION_UPDATE_FAILURE = 'CERTIFICATION_UPDATE_FAILURE';
export const CERTIFICATION_UPDATE_SUCCESS = 'CERTIFICATION_UPDATE_SUCCESS';

export const ON_CONTINUE_CLICK_FAILED = 'ON_CONTINUE_CLICK_FAILED';
export const ON_CONTINUE_CLICK_SUCCESS = 'ON_CONTINUE_CLICK_SUCCESS';

export const RESET_STATE = 'RESET_STATE';

// types of hearings
//
// TODO:
// on the backend, NO_HEARING_DESIRED
// should result in VIDEO being written to VACOLS,
// and HEARING_CANCELLED should result in a cancellation
// checkbox being checked, but the original hearing type
// should be undisturbed.
export const hearingPreferences = {
  VIDEO: 'VIDEO',
  TRAVEL_BOARD: 'TRAVEL_BOARD',
  WASHINGTON_DC: 'WASHINGTON_DC',
  HEARING_TYPE_NOT_SPECIFIED: 'HEARING_TYPE_NOT_SPECIFIED',
  NO_HEARING_DESIRED: 'NO_HEARING_DESIRED',
  HEARING_CANCELLED: 'HEARING_CANCELLED',
  NO_BOX_SELECTED: 'NO_BOX_SELECTED'
};

// form9 values
export const form9Types = {
  FORMAL_FORM9: 'FORMAL_FORM9',
  INFORMAL_FORM9: 'INFORMAL_FORM9'
};

// representation for the appellant
export const representativeTypes = {
  ATTORNEY: 'ATTORNEY',
  AGENT: 'AGENT',
  ORGANIZATION: 'ORGANIZATION',
  NONE: 'NONE',
  // TODO: should "Other be a real type"?
  OTHER: 'OTHER'
};

// was a hearing document found in VBMS?
export const vbmsHearingDocument = {
  FOUND: 'FOUND',
  NOT_FOUND: 'NOT_FOUND'
};

export const progressBarSections = {
  CHECK_DOCUMENTS: 'CHECK_DOCUMENTS',
  CONFIRM_CASE_DETAILS: 'CONFIRM_CASE_DETAILS',
  CONFIRM_HEARING: 'CONFIRM_HEARING',
  SIGN_AND_CERTIFY: 'SIGN_AND_CERTIFY'
};

export const certifyingOfficialTitles = {
  DECISION_REVIEW_OFFICER: 'DECISION_REVIEW_OFFICER',
  RATING_SPECIALIST: 'RATING_SPECIALIST',
  VETERANS_SERVICE_REPRESENTATIVE: 'VETERANS_SERVICE_REPRESENTATIVE',
  CLAIMS_ASSISTANT: 'CLAIMS_ASSISTANT',
  OTHER: 'OTHER'
};
