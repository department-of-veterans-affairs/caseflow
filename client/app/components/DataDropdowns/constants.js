import _ from 'lodash';

import HEARING_ROOMS_LIST from '../../../constants/HEARING_ROOMS_LIST.json';

export const HEARING_ROOM_OPTIONS = _.map(HEARING_ROOMS_LIST, (value, key) => ({
  label: value.label,
  value: key.toString()
}));
