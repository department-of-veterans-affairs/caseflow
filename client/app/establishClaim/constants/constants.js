// actions
export const CHANGE_SPECIAL_ISSUE = 'CHANGE_SPECIAL_ISSUE';
