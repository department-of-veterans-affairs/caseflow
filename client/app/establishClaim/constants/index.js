// actions
export const CHANGE_SPECIAL_ISSUE = 'CHANGE_SPECIAL_ISSUE';
export const CHANGE_ESTABLISH_CLAIM_FIELD = 'CHANGE_ESTABLISH_CLAIM_FIELD';
export const TOGGLE_CANCEL_TASK_MODAL = 'TOGGLE_CANCEL_TASK_MODAL';
export const TRIGGER_LOADING = 'TRIGGER_LOADING';
export const CHANGE_CANCEL_FEEDBACK = 'CHANGE_CANCEL_FEEDBACK';
export const VALIDATE_CANCEL_FEEDBACK = 'VALIDATE_CANCEL_FEEDBACK';
export const REQUEST_CANCEL_FEEDBACK = 'REQUEST_CANCEL_FEEDBACK';
export const REQUEST_CANCEL_FEEDBACK_FAILURE = 'REQUEST_CANCEL_FEEDBACK_FAILURE';
export const INCREMENT_MODIFIER_ON_DUPLICATE_EP_ERROR = 'INCREMENT_MODIFIER_ON_DUPLICATE_EP_ERROR';

// constants
export const FULL_GRANT = 'Full Grant';
export const MODIFIER_OPTIONS = [
  '070',
  '071',
  '072',
  '073',
  '074',
  '075',
  '076',
  '077',
  '078',
  '079'
];

export const ARC_STATION_OF_JURISDICTION = '397';
