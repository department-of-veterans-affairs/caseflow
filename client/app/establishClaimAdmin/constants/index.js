export const SET_LOADING_STATE = 'SET_LOADING_STATE';
export const POPULATE_STUCK_TASKS = 'POPULATE_STUCK_TASKS';
