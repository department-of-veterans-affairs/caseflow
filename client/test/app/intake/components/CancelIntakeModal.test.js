import React from 'react';
import { render, screen } from '@testing-library/react';
import { Provider } from 'react-redux';
import userEvent from '@testing-library/user-event';
import { createStore, applyMiddleware } from 'redux';
import thunk from 'redux-thunk';
import rootReducer from 'app/queue/reducers';
import CancelIntakeModal from 'app/intake/components/CancelIntakeModal';
import { CANCELLATION_REASONS } from 'app/intake/constants';

jest.mock('redux', () => ({
  ...jest.requireActual('redux'),
  bindActionCreators: () => jest.fn().mockImplementation(() => Promise.resolve(true)),
}));

describe('CancelIntakeModal', () => {
  const defaultProps = {
    closeHandler: () => {},
    intakeId: '123 change?',
    clearClaimant: jest.fn().mockImplementation(() => Promise.resolve(true)),
    clearPoa: jest.fn().mockImplementation(() => Promise.resolve(true)),
    submitCancel: jest.fn().mockImplementation(() => Promise.resolve(true))
  };
  const buttonText = 'Cancel intake';

  afterEach(() => {
    jest.clearAllMocks();
  });

  const store = createStore(rootReducer, applyMiddleware(thunk));

  const setup = (props) =>
    render(
      <Provider store={store}>
        <CancelIntakeModal
          {...props}
        />
      </Provider>
    );

  it('renders correctly', () => {
    const modal = setup(defaultProps);

    expect(modal).toMatchSnapshot();
  });

  it('displays cancellation options', () => {
    const modal = setup(defaultProps);

    Object.values(CANCELLATION_REASONS).map((reason) => (
      expect(modal.getByText(reason.name)).toBeInTheDocument()
    ));
  });

  it('should show other reason input when other is selected', async () => {
    const modal = setup(defaultProps);

    await userEvent.click(screen.getByText('Other'));

    expect(modal.getByText('Tell us more about your situation.')).toBeInTheDocument();
  });

  describe('cancel button', () => {
    it('is disabled until "Other" is selected and the text input is filled out', async () => {
      const modal = setup(defaultProps);

      expect(modal.getByText(buttonText)).toBeDisabled();

      await userEvent.click(modal.getByText('Other'));

      expect(modal.getByText('Tell us more about your situation.')).toBeInTheDocument();
      expect(modal.getByText(buttonText)).toBeDisabled();

      await userEvent.type(modal.getByRole('textbox'), 'Test');

      expect(modal.getByText(buttonText)).not.toBeDisabled();
    });

    it('is disabled until value (that is not "Other") is selected', async () => {
      const modal = setup(defaultProps);

      expect(modal.getByText(buttonText)).toBeDisabled();

      await userEvent.click(modal.getByText('System error'));
      expect(modal.getByText(buttonText)).not.toBeDisabled();
    });
  });

  it('should call the appropiate functions when Cancel intake is clicked', async () => {
    const modal = setup(defaultProps);

    await userEvent.click(modal.getByText('System error'));
    expect(modal.getByText(buttonText)).not.toBeDisabled();

    await userEvent.click(modal.getByText('Cancel intake'));
    expect(defaultProps.clearClaimant).toHaveBeenCalled();
    expect(defaultProps.clearPoa).toHaveBeenCalled();
    expect(defaultProps.submitCancel).toHaveBeenCalled();
  });
});
