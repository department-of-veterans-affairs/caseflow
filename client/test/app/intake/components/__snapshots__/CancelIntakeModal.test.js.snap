// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CancelIntakeModal renders correctly 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        class="intake-cancel"
      >
        <section
          aria-describedby="modal_id-desc"
          aria-labelledby="modal_id-title"
          aria-modal="true"
          class="cf-modal active "
          id="modal_id"
          role="dialog"
        >
          <div
            class="cf-modal-body"
            id=""
          >
            <button
              class="cf-modal-close"
              id="Cancel-Intake?-button-id-close"
              type="button"
            >
              <span
                class="usa-sr-only"
              >
                Close
              </span>
              <svg
                class="cf-icon-close"
                height="55"
                viewBox="0 0 55 55"
                xmlns="http://www.w3.org/2000/svg"
              >
                <title>
                  close
                </title>
                <g
                  fill="#212121"
                >
                  <path
                    d="M52.6 46.9l-6 6c-.8.8-1.9 1.2-3 1.2s-2.2-.4-3-1.2l-13-13-13 13c-.8.8-1.9 1.2-3 1.2s-2.2-.4-3-1.2l-6-6c-.8-.8-1.2-1.9-1.2-3s.4-2.2 1.2-3l13-13-13-13c-.8-.8-1.2-1.9-1.2-3s.4-2.2 1.2-3l6-6c.8-.8 1.9-1.2 3-1.2s2.2.4 3 1.2l13 13 13-13c.8-.8 1.9-1.2 3-1.2s2.2.4 3 1.2l6 6c.8.8 1.2 1.9 1.2 3s-.4 2.2-1.2 3l-13 13 13 13c.8.8 1.2 1.9 1.2 3s-.4 2.2-1.2 3z"
                  />
                </g>
              </svg>
            </button>
            <div
              style="display: flex;"
            >
              <div
                data-css-1gs0ko2=""
              >
                <h1
                  id="modal_id-title"
                >
                  Cancel Intake?
                </h1>
                <div
                  data-css-o98ubo=""
                >
                  <fieldset
                    class="usa-fieldset-inputs cf-form-radio "
                  >
                    <legend
                      class=""
                    >
                      <strong>
                        <span>
                          Please select the reason you are canceling this intake.
                           
                        </span>
                      </strong>
                    </legend>
                    <div
                      class="cf-form-radio-options"
                    >
                      <div
                        class="cf-form-radio-option"
                      >
                        <input
                          id="Please-select-the-reason-you-are-canceling-this-intake-_duplicate_ep"
                          name="Please select the reason you are canceling this intake."
                          type="radio"
                          value="duplicate_ep"
                        />
                        <label
                          class=""
                          for="Please-select-the-reason-you-are-canceling-this-intake-_duplicate_ep"
                        >
                          Duplicate EP created outside Caseflow
                        </label>
                      </div>
                      <div
                        class="cf-form-radio-option"
                      >
                        <input
                          id="Please-select-the-reason-you-are-canceling-this-intake-_system_error"
                          name="Please select the reason you are canceling this intake."
                          type="radio"
                          value="system_error"
                        />
                        <label
                          class=""
                          for="Please-select-the-reason-you-are-canceling-this-intake-_system_error"
                        >
                          System error
                        </label>
                      </div>
                      <div
                        class="cf-form-radio-option"
                      >
                        <input
                          id="Please-select-the-reason-you-are-canceling-this-intake-_missing_signature"
                          name="Please select the reason you are canceling this intake."
                          type="radio"
                          value="missing_signature"
                        />
                        <label
                          class=""
                          for="Please-select-the-reason-you-are-canceling-this-intake-_missing_signature"
                        >
                          Missing signature
                        </label>
                      </div>
                      <div
                        class="cf-form-radio-option"
                      >
                        <input
                          id="Please-select-the-reason-you-are-canceling-this-intake-_veteran_clarification"
                          name="Please select the reason you are canceling this intake."
                          type="radio"
                          value="veteran_clarification"
                        />
                        <label
                          class=""
                          for="Please-select-the-reason-you-are-canceling-this-intake-_veteran_clarification"
                        >
                          Need clarification from Veteran
                        </label>
                      </div>
                      <div
                        class="cf-form-radio-option"
                      >
                        <input
                          id="Please-select-the-reason-you-are-canceling-this-intake-_other"
                          name="Please select the reason you are canceling this intake."
                          type="radio"
                          value="other"
                        />
                        <label
                          class=""
                          for="Please-select-the-reason-you-are-canceling-this-intake-_other"
                        >
                          Other
                        </label>
                      </div>
                    </div>
                  </fieldset>
                </div>
              </div>
            </div>
            <div
              class="cf-modal-divider"
            />
            <div
              class="cf-modal-controls"
            >
              <span>
                <button
                  class="cf-modal-link cf-btn-link close-modal cf-push-left usa-button"
                  id="Cancel-Intake?-button-id-0"
                  type="button"
                >
                  Close
                </button>
              </span>
              <span>
                <button
                  class="confirm-cancel cf-push-right usa-button-disabled usa-button"
                  disabled=""
                  id="Cancel-Intake?-button-id-1"
                  type="button"
                >
                  Cancel intake
                </button>
              </span>
            </div>
          </div>
        </section>
        ;
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="intake-cancel"
    >
      <section
        aria-describedby="modal_id-desc"
        aria-labelledby="modal_id-title"
        aria-modal="true"
        class="cf-modal active "
        id="modal_id"
        role="dialog"
      >
        <div
          class="cf-modal-body"
          id=""
        >
          <button
            class="cf-modal-close"
            id="Cancel-Intake?-button-id-close"
            type="button"
          >
            <span
              class="usa-sr-only"
            >
              Close
            </span>
            <svg
              class="cf-icon-close"
              height="55"
              viewBox="0 0 55 55"
              xmlns="http://www.w3.org/2000/svg"
            >
              <title>
                close
              </title>
              <g
                fill="#212121"
              >
                <path
                  d="M52.6 46.9l-6 6c-.8.8-1.9 1.2-3 1.2s-2.2-.4-3-1.2l-13-13-13 13c-.8.8-1.9 1.2-3 1.2s-2.2-.4-3-1.2l-6-6c-.8-.8-1.2-1.9-1.2-3s.4-2.2 1.2-3l13-13-13-13c-.8-.8-1.2-1.9-1.2-3s.4-2.2 1.2-3l6-6c.8-.8 1.9-1.2 3-1.2s2.2.4 3 1.2l13 13 13-13c.8-.8 1.9-1.2 3-1.2s2.2.4 3 1.2l6 6c.8.8 1.2 1.9 1.2 3s-.4 2.2-1.2 3l-13 13 13 13c.8.8 1.2 1.9 1.2 3s-.4 2.2-1.2 3z"
                />
              </g>
            </svg>
          </button>
          <div
            style="display: flex;"
          >
            <div
              data-css-1gs0ko2=""
            >
              <h1
                id="modal_id-title"
              >
                Cancel Intake?
              </h1>
              <div
                data-css-o98ubo=""
              >
                <fieldset
                  class="usa-fieldset-inputs cf-form-radio "
                >
                  <legend
                    class=""
                  >
                    <strong>
                      <span>
                        Please select the reason you are canceling this intake.
                         
                      </span>
                    </strong>
                  </legend>
                  <div
                    class="cf-form-radio-options"
                  >
                    <div
                      class="cf-form-radio-option"
                    >
                      <input
                        id="Please-select-the-reason-you-are-canceling-this-intake-_duplicate_ep"
                        name="Please select the reason you are canceling this intake."
                        type="radio"
                        value="duplicate_ep"
                      />
                      <label
                        class=""
                        for="Please-select-the-reason-you-are-canceling-this-intake-_duplicate_ep"
                      >
                        Duplicate EP created outside Caseflow
                      </label>
                    </div>
                    <div
                      class="cf-form-radio-option"
                    >
                      <input
                        id="Please-select-the-reason-you-are-canceling-this-intake-_system_error"
                        name="Please select the reason you are canceling this intake."
                        type="radio"
                        value="system_error"
                      />
                      <label
                        class=""
                        for="Please-select-the-reason-you-are-canceling-this-intake-_system_error"
                      >
                        System error
                      </label>
                    </div>
                    <div
                      class="cf-form-radio-option"
                    >
                      <input
                        id="Please-select-the-reason-you-are-canceling-this-intake-_missing_signature"
                        name="Please select the reason you are canceling this intake."
                        type="radio"
                        value="missing_signature"
                      />
                      <label
                        class=""
                        for="Please-select-the-reason-you-are-canceling-this-intake-_missing_signature"
                      >
                        Missing signature
                      </label>
                    </div>
                    <div
                      class="cf-form-radio-option"
                    >
                      <input
                        id="Please-select-the-reason-you-are-canceling-this-intake-_veteran_clarification"
                        name="Please select the reason you are canceling this intake."
                        type="radio"
                        value="veteran_clarification"
                      />
                      <label
                        class=""
                        for="Please-select-the-reason-you-are-canceling-this-intake-_veteran_clarification"
                      >
                        Need clarification from Veteran
                      </label>
                    </div>
                    <div
                      class="cf-form-radio-option"
                    >
                      <input
                        id="Please-select-the-reason-you-are-canceling-this-intake-_other"
                        name="Please select the reason you are canceling this intake."
                        type="radio"
                        value="other"
                      />
                      <label
                        class=""
                        for="Please-select-the-reason-you-are-canceling-this-intake-_other"
                      >
                        Other
                      </label>
                    </div>
                  </div>
                </fieldset>
              </div>
            </div>
          </div>
          <div
            class="cf-modal-divider"
          />
          <div
            class="cf-modal-controls"
          >
            <span>
              <button
                class="cf-modal-link cf-btn-link close-modal cf-push-left usa-button"
                id="Cancel-Intake?-button-id-0"
                type="button"
              >
                Close
              </button>
            </span>
            <span>
              <button
                class="confirm-cancel cf-push-right usa-button-disabled usa-button"
                disabled=""
                id="Cancel-Intake?-button-id-1"
                type="button"
              >
                Cancel intake
              </button>
            </span>
          </div>
        </div>
      </section>
      ;
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
